crm_win_reason/models/crm_lost_reason.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class CrmLostReason(models.Model):
+    _inherit = "crm.lost.reason"
+
+    reason_type = fields.Selection(
+        [("won", "Won"), ("lost", "Lost")],
```

Comment:
On the wizard, the domain is hardcoded to either `[('reason_type','=','won')]` or `[('reason_type','=','lost')]`.
So, either:

- Make it required here
- Allow for non-specifcied reason types (global // apply to both won or lost):
    ie: `[('reason_type','in',[False, 'won'])]`

Reply:
@ivantodorovich @TDu  thank you, comments from review addressed